### PR TITLE
fix: Rework import

### LINKED ISSUE #120: change default import folder
by default the import command tries to import from ./import folder, that has no practical use, more since currently the cwd is the folder where the assets are (usually the extracted folder with the binary and assets) 

it is better to not have any default and request to pass a folder explicitly, or use `.` current working directory, which is not good right now for the previously mentioned issue of cwd needing to be the folder where the assets folder is

### PR DESCRIPTION
**Pull request by Septias**
- create module for import
- allow single file import as well as multifile
- Use cwd as default directory

close #120

### CODE REVIEW

File: src/import.rs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    let mut app_info = AppInfo::from_xdc(file).await?;
+    app_info.active = true;
+    app_info.submitter_uri = Some("xdcstore".to_string());
+
+    // copy the file to the `dest`
+    let mut dest = PathBuf::from(dest);
+    dest.push(file.file_name().context("Direntry has no filename")?);
+    fs::copy(file, &dest)
+        .with_context(|| format!("Failed to copy {} to {}", file.display(), dest.display()))?;
+    app_info.xdc_blob_dir = dest;
```
Comment (hpk42) [inline src/import.rs:63]: hope this gets renamed soon because it's not a dir ;)

**Review by hpk42 — APPROVED** [review]
looks good but please add a test to call at least the dir-function using the example-xdc's

**Comment by Septias** [comment]
> looks good but please add a test to call at least the dir-function using the example-xdc's

This is done plenty of times in the python tests

**Comment by hpk42** [comment]
On Thu, Jun 29, 2023 at 08:50 -0700, Sebastian Klähn wrote:
> > looks good but please add a test to call at least the dir-function using the example-xdc's
> 
> This is done plenty of times in the python tests

ah right, that's probably enough then :)

**Review by adbenitez — APPROVED** [review]

**Pull request merged**